reducers/reducers.js

```diff
@@ -312,7 +313,14 @@ function appVersion(state = "", action) {
       return state;
   }
 }
-
+function startBani(state=JSON.stringify([]),action){
+  switch(action.type){
+    case SET_START_BANI:
+      return action.progressList
+    default:
+      return state
+  }
+}
```

Comment:
Veerji it will be better if we stick to the old indentation of additional space before the curly brace { starts after the function parameters. It will keep maintaining the standard. I would recommend you to use eslint or something similar and setup it up according to existing indentation and linting rules.

For instance, you can compare line 316 with line 38.

Reply:
Fixed